src/components/Counter/Counter.jsx

```diff
@@ -1,9 +1,33 @@
-import { useEffect, useState } from 'react'
+import { useEffect, useState, useReducer } from 'react'
 
 const pinkRGB = `rgb(236, 72, 153)`
+const initialCount = 0
+const ACTIONS = {
+  INCREMENT: 'increment',
+  DECREMENT: 'decrement',
+  RESET: 'reset'
+}
+
+function countReducer(count, action) {
+  switch (action.type) {
+    case ACTIONS.INCREMENT: {
```

Comment:
You can also just name these cases what you've named the Increments on 6-8 to save writing that piece. 'case 'increment: {other code}'